homeassistant/components/matter/select.py

```diff
@@ -435,4 +435,65 @@ def _update_from_device(self) -> None:
         # don't discover this entry if the supported rinses list is empty
         secondary_value_is_not=[],
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=MatterSelectEntityDescription(
+            key="LocalConfigDisabled",
+            entity_category=EntityCategory.CONFIG,
+            translation_key="local_config_disabled",
+            options=["Enable Local Configuration", "Disable Local Configuration"],
```

Comment:
```suggestion
            options=["enable", "disable"],
```
Use strings.json to map this to human readable name.
That said: why is this a select entity and not a switch if its just a toggle ?